MachObfuscator/SymbolMangling/SymbolManglers.swift

```diff
@@ -1,11 +1,13 @@
 import Foundation
 
 enum SymbolManglers {
-    static let allManglers: [SymbolMangling.Type] = [CezarMangler.self, RealWordsMangler.self]
+    static let allManglers: [SymbolMangling] = [CaesarMangler(exportTrieMangler: CaesarExportTrieMangler()),
+                                                RealWordsMangler(exportTrieMangler: RealWordsExportTrieMangler())]
+
+    static let defaultManglerKey = "realWords"
```

Comment:
I am glad you moved this const out of `Options`, but I am still worried about using `"realWords"` string literal here, while it is used in `RealWordsMangler` as:
```
   var key: String = "realWords"
```
When one of those strings changes there is no unit test that detects that. I think we should have one const that will be used in both places.